tests/integration/conftest.py

```diff
@@ -428,9 +428,10 @@ def runtime_ctx(ws, sql_backend, make_table, make_schema, make_udf, make_group,
 
 
 class LocalAzureCliTest(WorkspaceContext):
```

Comment:
What if you extend the TestRuntimeContext or TestInstallationContext, so that you don't have to copy-paste so many cached properties?..